sunkit_pyvista/tests/test_pyvista.py

```diff
@@ -97,3 +100,31 @@ def test_clip_interval(aia171_test_map, plotter):
     with pytest.raises(ValueError, match=r"Clip percentile interval must be "
                        r"specified as two numbers."):
         plotter.plot_map(aia171_test_map, clip_interval=(1, 50, 99)*u.percent)
+
+
+def test_multi_block(plotter):
+    plotter.plot_solar_axis()
+    assert plotter.all_meshes['solar_axis'][0].n_cells == 43
+    assert plotter.all_meshes['solar_axis'][0].n_points == 101
+
+
+def test_field_lines(aia171_test_map, plotter):
+    gong_fname = get_gong_map()
+    gong_map = smap.Map(gong_fname)
+    nrho = 35
+    rss = 2.5
+    lat = np.linspace(-np.pi / 2, np.pi / 2, 8, endpoint=False)
+    lon = np.linspace(0, 2 * np.pi, 8, endpoint=False)
+    lat, lon = np.meshgrid(lat, lon, indexing='ij')
+    lat, lon = lat.ravel() * u.rad, lon.ravel() * u.rad
+    radius = 1.2
+    tracer = tracing.PythonTracer()
+    input_ = pfsspy.Input(gong_map, nrho, rss)
+    output_ = pfsspy.pfss(input_)
+    seeds = SkyCoord(lon, lat, radius*const.R_sun,
+                     frame=gong_map.coordinate_frame)
+    field_lines = tracer.trace(seeds, output_)
+    plotter.plot_field_lines(field_lines)
+    assert len(plotter.all_meshes['field_lines'][0]) == 64
+    assert isinstance(plotter.all_meshes['field_lines'][0],
+                      pyvista.core.composite.MultiBlock)
```

Comment:
```suggestion
                      pv.core.composite.MultiBlock)
```
Should fix the tests

Reply:
Ah, my bad, sorry